plugins/GSdx/GSSetting.cpp

```diff
@@ -55,7 +55,9 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_ALPHAHACK:
 			return "Different alpha handling. Can work around some shadow problems.";
 		case IDC_OFFSETHACK:
-			return "Might fix some misaligned fog, bloom, or blend effect.";
+			return "Might fix some misaligned fog, bloom, or blend effect.\n"
+				"Preferred option should be Normal (Vertex). Special cases are only useful in couple of games like Captain Tsubasa.\n"
```

Comment:
It'd probably be better to expend it to:
"The preferred option is Normal (Vertex) as it is most likely to resolve misalignment issues. The special...